src/calculator/service.ts

```diff
@@ -156,9 +156,17 @@ async function getUserInfo(userId: string) : Promise<TwitterUser> {
 }
 
 async function getTweetInfo(tweetId: string) : Promise<Tweet> {
-  const client = buildTwitClient()
+  const DEBUG_TWEET_TWITTER_API_TIME_LABEL = 'Time spent calling the Twitter API\
+  to get tweet info'
+  
+  console.log('1')
+  console.time(DEBUG_TWEET_TWITTER_API_TIME_LABEL)
+  const client = buildTwitClient()  
+
   try {
     const response = await client.get('statuses/show', { id: tweetId, tweet_mode: 'extended' })
+    console.timeEnd(DEBUG_TWEET_TWITTER_API_TIME_LABEL)
+    
     return responseToTweet(response.data)
   } catch (e) {
     console.log(e)
```

Comment:
Please `timeEnd` the timer here as well
```suggestion
    console.log(e)
    console.timeEnd(DEBUG_TWEET_TWITTER_API_TIME_LABEL)
```

Reply:
...otherwise, we will get weird timer results before it is not closing.

Actually here thinking, We should add a _random_ garbage to the end of that string to avoid concurrent requests to affect the timer. Something as simple as using `Math.random()` will be enough I think.